package.json

```diff
@@ -45,8 +44,6 @@
     "escape-html": "^1.0.3",
     "lodash.defaultsdeep": "^4.6.1",
```

Comment:
I think `lodash.defaultsdeep` can be dropped too in favor `{ ...options }` syntax

Reply:
There are a few of nested fields, so it is not that straightforward.

Reply:
We can do nested clone, i.e. `{ foo: { ...options.foo } }`